welcome/openshift-editions.adoc

```diff
@@ -0,0 +1,47 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="openshift-editions"]
+= Red Hat OpenShift editions
+include::_attributes/common-attributes.adoc[]
+:context: openshift-editions
+
+toc::[]
+
+Red Hat OpenShift is offered in several editions to support a wide range of deployment models and operational preferences. Each edition delivers a consistent Kubernetes platform with integrated tools, security features, and developer experiences. OpenShift is available in cloud services and self-managed editions.
+
+[id="cloud-services-editions_{context}"]
+== Cloud services editions
+
+Red Hat OpenShift offers various cloud service editions to cater to different organizational needs. These editions provide fully managed application platforms from major cloud providers.
+
+{{product-rosa}} (ROSA):: A fully managed application platform that helps organizations build, deploy, and scale applications in a native AWS environment.
+For more information, see link:https://www.redhat.com/en/technologies/cloud-computing/openshift/aws[{{product-rosa}}].
+
+{{azure-first}} Red Hat OpenShift:: A fully managed application platform that helps organizations build, deploy, and scale applications on Azure.
+For more information, see link:https://www.redhat.com/en/technologies/cloud-computing/openshift/azure[{{azure-first}} Red Hat OpenShift].
```

Comment:
```suggestion
For more information, see link:https://www.redhat.com/en/technologies/cloud-computing/openshift/azure[{azure-first} Red{nbsp}Hat OpenShift].
```

`{nbsp}` can be added in this module as per 
https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

> Use a non-breaking space ({nbsp}) between the words "Red" and "Hat"...